src/FapiApi.php

```diff
@@ -30,7 +30,7 @@ public function __construct( $apiUser, $apiKey, $apiUrl = 'https://api.fapi.cz/'
 	 * @param int $id
 	 * @return false|array<mixed>
 	 */
-	public function getInvoice( $id ) {
+	public function getInvoice( $id = '' ) {
```

Comment:
Tohle způsobí přepsání proměnné a tedy nenačtení žádné faktury, opravit prosím.

Reply:
Toto je myslené ako default hodnota, t.j. keď je funkcia zavolaná bez parametra $id, nastaví sa na prázdny string čo spôsobí, že to vráti všetky faktúry pre všetky prepojené účty, testoval som to.